mne/decoding/tests/test_search_light.py

```diff
@@ -254,6 +254,33 @@ def test_generalization_light():
     assert_array_equal(y_preds[0], y_preds[1])
 
 
+@requires_sklearn
+def test_verbose_arg(capsys):
+    """Test controlling output with the ``verbose`` argument."""
+    from sklearn.svm import SVC
+
+    X, y = make_data()
+    clf = SVC()
+
+    # shows progress bar and prints other messages to the console
+    with use_log_level(True):
```

Comment:
Worth doing a `@pytest.mark.parametrize('orig_log_level', (True, False))`?

Reply:
I copied that from Progressbar tests, I'm not sure the test works without that on CIs...